airflow/providers/amazon/aws/example_dags/example_ecs.py

```diff
@@ -99,18 +100,20 @@
     )
     # [END howto_operator_ecs_register_task_definition]
 
+    registered_task_definition = cast(str, register_task.output)
```

Comment:
It’s probably a good idea to add a Mapy plugin for this… I’ll look into it after this is merged.

Reply:
Great! I was going to ask if there was a better way than doing all of these `cast()` calls. Had to be something we could do with mypy specifically.